### PR TITLE
Registration page

### PR DESCRIPTION
**Pull request by piet0024**
Modified the landing page to a registration form.

### CODE REVIEW

File: src/pages/ProcessEn.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const ProcessEn = () => (
+  <main className="text-center splash-page process-splash">
+    <Helmet>
+      <title>Processing Request</title>
+    </Helmet>
+    <div className="process-holder form-holder">
+      <img className="gcx-logo" src={gcxLogo} alt="gcxchange | gcéchange" />
+      <h1 className="process-title">Sit tight, we’re almost finished setting up your account! You’ll be getting an email from us soon. </h1>
+      <img className="info-img" src={processingImage} alt="" />
+      <p>If you don't receive an email after 10 minutes, please contact us at <a href="mailto:support-soutien@gcx-gce.gc.ca.">support-soutien@gcx-gce.gc.ca. </a></p>
```
Comment (Steph4104) [inline src/pages/ProcessEn.js:16]: remove the . in the href mailto

File: src/pages/ProcessFr.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const ProcessFr = () => (
+  <main lang="fr" className="text-center splash-page process-splash">
+    <Helmet>
+      <title>Processing Request</title>
+    </Helmet>
+    <div className="process-holder form-holder">
+      <img className="gcx-logo" src={gcxLogo} alt="gcxchange | gcéchange" />
+      <h1 className="process-title">Attendez patiemment, nous avons presque fini de configurer votre compte! Vous recevrez bientôt un courriel de notre part.</h1>
+      <img className="info-img" src={processingImage} alt="" />
+      <p>Si vous ne recevez pas de courriel après 10 minutes, veuillez communiquer avec nous à l’adresse suivante : <a href="mailto:support-soutien@gcx-gce.gc.ca.">support-soutien@gcx-gce.gc.ca.</a></p>
```
Comment (Steph4104) [inline src/pages/ProcessFr.js:16]: remove the . in the href mailto

File: src/i18n/lang.js [hunk 3]
```diff
@@ -2,39 +2,30 @@ const Lang = {
...
+            list4: '<span class="info-highlight">Share</span> your content with the GC community',
             link: process.env.REACT_APP_ACCESS_PACKAGE_EN,
         },
-        step1: {
-            heading: 'Step <span class="highlight">one</span>',
-            paragraph1: '<strong>Check off</strong> the circle beside “Get access to gcxchange.”<br/> Then, click <strong>+ Request access</strong>.'
+        form: {
+            loading: 'Loading, please wait',
+            emailLabel: 'Departmental email address (Required) ',
+            emailPlaceholder: 'john.smith@tbs-sct.gc.ca, john.smith@ssc-spc.gc.ca, etc',
```
Comment (Steph4104) [inline src/i18n/lang.js:16]: Remove etc

File: src/i18n/lang.js [hunk 4]
```diff
@@ -47,39 +38,30 @@ const Lang = {
...
-            paragraph3: 'Toutefois, attendez avant de cliquer — le système a besoin de quelques minutes (au plus cinq minutes) pour terminer l’activation. La patience est mère de toutes les vertus, et la récompense, c’est que lorsque vous cliquerez finalement sur le lien, vous y serez déjà.',
-            paragraph4: 'Si vous recevez le message « Ça n’a pas marché » après avoir cliqué sur le lien, ne paniquez pas. Le système a tout simplement besoin d’un peu plus de temps pour activer votre compte. Attendez encore quelques minutes, puis actualisez la page.',
-            paragraph5: 'Si vous avez besoin d’aide entre-temps, n’hésitez pas à communiquer avec notre <a href="mailto:solution-soutien@gcx-gce.gc.ca">équipe de soutien</a>.',
+        form: {
+            loading: 'chargement, veuillez patienter',
+            emailLabel: 'Adresse courriel du ministère (Obligatoire) ',
+            emailPlaceholder: 'jeanne.tremblay@tbs-sct.gc.ca, jeanne.tremblay@ssc-spc.gc.ca, etc.',
```
Comment (Steph4104) [inline src/i18n/lang.js:52]: Remove etc

**Review by Steph4104 — CHANGES_REQUESTED** [review]
When I enter my email the register button switch to enable but the department field is empty. I did the testing in edge.

Is it possible to have the department dropdown in alphabetical order for french and english.

**Review by Phanoix — APPROVED** [review]

**Review by ethanWallace — APPROVED** [review]

**Pull request merged**